dictionaries/ui/pages/preferences/en.dictionary.itop.preferences.php

```diff
@@ -24,6 +24,8 @@
 	'UI:Preferences:General:Title'                                     => 'General',
 	'UI:Preferences:General:Theme'                                     => 'Theme',
 	'UI:Preferences:General:Theme:DefaultThemeLabel'                   => '%1$s (default)',
+	'UI:Favorites:General:ShowSummaryCard'                          => 'Show summary cards',
+	'UI:Favorites:General:ShowSummaryCard+'                         => 'When hovering an hyperlink targeting an object, display a short summary of the object',
```

Comment:
```suggestion
	'UI:Favorites:General:ShowSummaryCards'                          => 'Show summary cards',
	'UI:Favorites:General:ShowSummaryCards+'                         => 'When hovering an hyperlink targeting an object, display a short summary of the object',
```

Here it's ok to keep "summary cards" instead of "object summary" IMO as this feature (display content on hover of something) could also be used for other things in the future.